test/integration/download-tests.js

```diff
@@ -42,17 +42,23 @@ describe('Firefox Send', function() {
     );
     browser.waitForExist(homePage.uploadButton);
     browser.waitForExist(homePage.downloadCountSelect);
+    // Select 2 downloads
     browser.selectByIndex(homePage.downloadCountSelect, 1);
     browser.click(homePage.uploadButton);
+    // TODO: need something like `await browser.waitForExist(homePage.shareUrl)`
     browser.waitForExist(homePage.shareUrl);
+    // Newly added file will always be first in the list
+    const archiveId = browser.getAttribute('send-archive', 'id')[0];
     const downloadPage = new DownloadPage(browser.getValue(homePage.shareUrl));
+    downloadPage.setUseCase(archiveId, 'CHECK_DL_COUNT');
     downloadPage.open();
     downloadPage.download();
     browser.waitForExist(downloadPage.downloadComplete);
     browser.back();
-    browser.waitForExist('send-archive');
+    browser.waitForExist(`#${archiveId}`);
+    const uploadedFile = downloadPage.getUploadedFiles('CHECK_DL_COUNT')[0];
     assert.equal(
-      browser.getText('send-archive > div').substring(0, 24),
```

Comment:
This selector broke when we added another child div for the download link. All that needs to happen is change it to `send-archive > div:first-of-type`. If you'd rather not do it here we could also add it as another property in `pages/desktop/download_page.js`